### PR TITLE
added more war events

### PR DESCRIPTION
**Pull request by dusk0220**
added a plethora of sacrifice/accidental death events

### CODE REVIEW

File: resources/dicts/events/death/general/general.json [hunk 1]
```diff
@@ -273,5 +273,108 @@
...
+        "camp": "any",
+        "tags": [
+            "Newleaf",
+            "Greenleaf",
+            "Leaf-fall",
+            "Leaf-bare",
+            "war",
+            "other_cat"
+        ],
+        "death_text": "m_c sacrificed themself for r_c in a battle with o_c.",
```
Comment (scribblecrumb) [inline resources/dicts/events/death/general/general.json]: missing a pronoun tag for "themself", double check pronoun tags for all your additions please!

File: resources/dicts/events/misc_events/general/mediator.json [hunk 2]
```diff
@@ -34,5 +34,37 @@
         "cat_skill": null,
         "other_cat_trait": null,
         "other_cat_skill": null
+    },
+    {
+        "camp": "any",
+        "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "war", "other_cat_leader" ],
```
Comment (scribblecrumb) [inline resources/dicts/events/misc_events/general/mediator.json]: make sure to add the "other_cat" as well!  the more specific "other_cat_leader" tags will not work without it

File: resources/dicts/events/new_cat/general/general.json [hunk 3]
```diff
@@ -324,5 +324,41 @@
...
+        "backstory": ["otherclan2", "disgraced", "refugee1", "tragedy_survivor1"],
+        "new_name": false
+    },
+    {
+        "camp": "any",
+        "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare"],
+        "reputation": ["hostile"],
+        "event_text": "A member of o_c defects from their birth Clan and chooses to join c_n. They have decided to keep their name.",
+        "other_clan": true,
+        "backstory": ["otherclan2", "disgraced", "refugee1", "tragedy_survivir1", "otherclan2"],
```
Comment (scribblecrumb) [inline resources/dicts/events/new_cat/general/general.json]: mispelled tragedy_survivor

**Review by scribblecrumb — CHANGES_REQUESTED** [review]
Looks great!  Just a few little fixes and missing pronoun tags.  I'd also love to see some of these death events have corresponding injury events where the cat survives, but with injuries!  But, that of course can be a different PR.

File: resources/dicts/events/death/general/general.json [hunk 4]
```diff
@@ -347,7 +347,7 @@
             "Leaf-bare",
             "war"
         ],
-        "death_text": "m_c sacrificed themselves by taking on the role of a messenger, delivering important information that could save their Clan. They were killed while investigating enemy territory.",
+        "death_text": "m_c sacrificed {PRONOUN/m_c/self} by taking on the role of a messenger, delivering important information that could save {PRONOUN/m_c/poss} Clan. {PRONOUN/m_c/subject} were killed while investigating enemy territory.",
```
Comment (scribblecrumb) [inline resources/dicts/events/death/general/general.json]: "were" in the last sentence needs to be a VERB tag

File: resources/dicts/events/new_cat/general/general.json [hunk 5]
```diff
@@ -329,7 +329,7 @@
         "camp": "any",
         "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "war"],
         "reputation": ["hostile"],
-        "event_text": "A member of o_c defects in the war and chooses to join c_n. They have decided to keep their name.",
+        "event_text": "A member of o_c defects in the war and chooses to join c_n. {PRONOUN/m_c/subject} have decided to keep {PRONOUN/m_c/poss} name.",
```
Comment (scribblecrumb) [inline resources/dicts/events/new_cat/general/general.json]: "have" needs to be a VERB tag

File: resources/dicts/events/new_cat/general/general.json [hunk 6]
```diff
@@ -338,16 +338,16 @@
...
-        "backstory": ["otherclan2", "disgraced", "refugee1", "tragedy_survivir1", "otherclan2"],
+        "backstory": ["otherclan2", "disgraced", "refugee1", "tragedy_survivor1", "otherclan2"],
         "new_name": false
     },
    {
         "camp": "any",
         "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "war"],
         "reputation": ["hostile"],
-        "event_text": "n_c, the deputy of o_c, has defected to c_n in the war. They decide to keep their name.",
+        "event_text": "n_c, the deputy of o_c, has defected to c_n in the war. {PRONOUN/m_c/subject} decide to keep {PRONOUN/m_c/poss} name.",
```
Comment (scribblecrumb) [inline resources/dicts/events/new_cat/general/general.json]: "decide" needs to be a VERB tag

**Review by scribblecrumb — CHANGES_REQUESTED** [review]
missed some VERB tags! make sure to double check the verbs by reading the sentence with both they/them and she/her or he/him pronouns in order to check that plural/singular verbs are correctly tagged.

**Review by Ryos00 — APPROVED** [review]

**Pull request merged**